### DISCUSSION TITLE
Description field in Allowed Amount file required or not required, ,discrepancy exists in read.me versus schema

### DISCUSSION THREAD

**Discussion by tdfow** [CMSgov/price-transparency-guide#29] [Q&A] [answered]
Is the Description field required or not required?   In the Read.me for Allowed Amount, it states required yet the schema does not have it as required.  Which is correct?

The In-Network Rate file has the description as not required.  My assumption is the description is not required for both--but information published has conflicting data.

Can this be updated to the appropriate expectation?

**Comment by shaselton-usds** [comment, marked as answer]
Thanks @tdfow. You've been amazing at catching these inconsistencies and I really appreciate it. The description field is initially going to be optional for in-network and going to be required for the allowed amount file. 

I'll update the schema.

**Reply by nmayle** [reply]
The language in the regulation seems to say the the description is required for every item:
 
   

> (B) A billing code, which in the case of prescription drugs must be an NDC, **and a plain language description for each billing code for each covered item or service** under each coverage option offered by a plan or issuer; and...


Where do you find language that says the description is optional?

There is also further requirements that the description be plain language and that is defined as: 

> Plain language means written and presented in a manner calculated to be understood by the average participant or beneficiary.

**Reply by tdfow** [reply]
My question was based on that the description being required in one file and not the other, so wanted confirmation for which direction was needed.  Since the description field is required for Allowed Amount, will there be any expectations for the plain language description to be consistently reported by all Plan/Issuers or will there be variations by reporting entity?  I know there are licenses concerns around the description information, so is there any guidance

**Reply by bryanbriegel** [reply]
the ICRs for the In Network and Allowed Amount files note "Brief description of the item or service. In the case of items and services that are associated with common billing codes (such as the HCPCS codes), the code’s associated short text description may be provided. In the case of NDCs for prescription drugs, the plain language description must be the proprietary and nonproprietary names assigned to the NDC by the Food and Drug Administration (FDA)."  The Prescription Drug file notes "The proprietary and nonproprietary name assigned to the National Drug Code (NDC) by the Food and Drug Administration (FDA)." - agree there are licensing considerations.
